### ISSUE TITLE
Add parallel logging to json

### ISSUE THREAD

**Issue by karandwivedi42** [lanpa/tensorboard-pytorch#136]
It would be useful to have a flag that enables logging to a json file in parallel to the events file. That way we can look at the raw data and plot it outside of tensorboard. Also, it will happen in parallel (compared to the `export_to_json`) so it will work even for crashed experiments.

Right now I have a wrapper lib for tensorboardX that first saves to json and then calls tensorboard. I can raise a PR if you want. It looks somewhat like this

```python
Class Logger:
	...
	def add_scalar(x,tag,gs):
		with open(self.log_file) as f:
			json.dump(...)
		self.tb.add_scalar(x,tag,gs)
```

**Comment by lanpa** [comment]
Hi, I think it should be part of main repo because:
1. I remember the data can be downloaded in csv format via tensorboard interface recently.
2. values can be dumped with additional `json.dump(...)` or `f.write(...)` before the original add_scalar.

**Comment by karandwivedi42** [comment]
1 is a bit impractical if you have a lot of graphs. 2 is what I am doing right now and was proposing to add as a flag within tensorboardX.

**Comment by lanpa** [comment]
should be `fixes #134` QQ
